plugins/vppv2/model/punt/punt.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package punt;
+
+/* L3 protocol definition */
+enum L3Protocol {
+    IPv4 = 0;
```

Comment:
Enums should provide _undefined_ as the default zero value to avoid any ambiguity.